simplic-user/Simplic.User.UI/Utils/IOpenSelectedUserDetails.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Windows.Input;
+
+namespace Simplic.User.UI.Utils
+{
+    interface IOpenSelectedUserDetails
+    {
+        ICommand OpenSelectedUserDetailsCommand
+        {
+            get;
+            set;
+        }
+    }
```

Comment:
Interfaces should have a comment in any way

Reply:
Done